### DISCUSSION TITLE
How to add a new chain network ?

### DISCUSSION THREAD

**Discussion by umershaikh123** [Uniswap/interface#7681] [Q&A]
Is there a way to add a new chain support to the uniswap ? Do I have to fork the repo and add it manually myself ?
like for example I want to use uniswap on polygonzkevm chain , how would that work ?.
it would be really helpful if you can guide me

**Comment by mikeghen** [comment]
I'd also like to see this because it doesn't seem like there's going to be enough support for testnets added quick enough to make the interface useful for developers. So much extra development time will be spent by devs scripting custom foundry scripts to do super basic interactions, like swap. I mentioned this in another #7900 where the lack of a second testnet significantly increased the amount of code that has to be written to test.
